docker-compose.override.yml

```diff
@@ -88,20 +88,20 @@ services:
       - >-
         traefik.http.routers.traefik-redirect.rule=Host(`georchestra-127-0-1-1.traefik.me`) && (
         Path(`/analytics`)
-        || PathPrefix(`/datafeeder`)
-        || PathPrefix(`/datahub`) 
-        || PathPrefix(`/console`) 
-        || PathPrefix(`/geonetwork`)
-        || PathPrefix(`/geoserver`)
-        || PathPrefix(`/header`)
-        || PathPrefix(`/import`)
-        || PathPrefix(`/mapstore`)
+        || Path(`/datafeeder`)
+        || Path(`/datahub`)
+        || Path(`/console`)
+        || Path(`/geonetwork`)
+        || Path(`/geoserver`)
+        || Path(`/header`)
+        || Path(`/import`)
+        || Path(`/mapstore`)
         )
       - "traefik.http.routers.traefik-redirect.priority=10"
       - "traefik.http.routers.traefik-redirect.middlewares=add-trailing-slash@docker"
       - "traefik.http.middlewares.add-trailing-slash.redirectregex.regex=^https?://(.*)/(.+)"
       - "traefik.http.middlewares.add-trailing-slash.redirectregex.replacement=https://$${1}/$${2}/"
-      - "traefik.http.middlewares.add-trailing-slash.redirectregex.permanent=true"
+      - "traefik.http.middlewares.add-trailing-slash.redirectregex.permanent=false"
```

Comment:
I don't get it on this change. "There is a simple difference between a 301 and 302 redirect: a 301 (permanent) redirect indicates that a page has permanently moved to a new location, meanwhile, a 302 redirect says that the page has moved to a new location, but that it is only temporary." 
In this case, there is no "temporary", hence 301 redirect should be used IMO.

Reply:
IMHO, it is more safe to use temporary redirect as far as this might change at some time (for example, like here, if there is a mistake, new component or change in routing).

If we set a permanent redirect all users will need to clear the cache, and we cannot inform all users to do so, and a lot of people might lost a lot of time.

So I would recommend keeping a temporary redirect as the redirect is done directly in the reverse proxy, the cost of a request and redirect might be less that a problem with bad permanent redirect.

Reply:
:+1: